syntax = "proto3";

package medicine;

service MedicineService {
  rpc GetMedicineById (MedicineIdRequest) returns (MedicineResponse);
}

message MedicineIdRequest {
  int32 medicine_id = 1;
}

message MedicineResponse {
  bool found = 1;
  int32 medicine_id = 2;
  string name = 3;
  string MFG = 4;
  string EXP = 5;
  string unit = 6;
  int32 price = 7;
  int32 amount = 8;
}